tensorflow/core/platform/default/build_config.bzl

```diff
@@ -8,7 +8,7 @@ load("//tensorflow:tensorflow.bzl", "if_not_mobile")
 WITH_GCP_SUPPORT = False
 WITH_HDFS_SUPPORT = False
 WITH_XLA_SUPPORT = False
-WITH_JEMALLOC = True
+WITH_JEMALLOC = False
```

Comment:
Can we make this specific to arm cross-builds if it's necessary? Doesn't seem like the sort of default we'd want to change and I can't imagine it being accepted upstream.